cross-arch-build.sh

```diff
@@ -0,0 +1,20 @@
+set -exu
+
+if [ "${TARGETPLATFORM}" = "linux/amd64" ]; then
+  RUST_TARGET=x86_64-unknown-linux-gnu
```

Comment:
These targets should be right - these are the MUSL target for rust, which is right for alpine linux (vs. other linux distros that have legit libc)